petastorm/transform.py

```diff
@@ -26,8 +26,9 @@ def __init__(self, func, edit_fields=None, removed_fields=None):
         ``edit_fields`` and ``removed_fields`` define mutating operations performed on the original schema that
         produce a post-transform schema. ``func`` return value must comply to this post-transform schema.
 
-        :param func: A callable. The function is called on the worker thread. It takes a dictionary that complies to
-          the input schema and must return a dictionary that complies to a post-transform schema.
+        :param func: Optional. A callable. The function is called on the worker thread. It takes a dictionary that
+          complies to the input schema and must return a dictionary that complies to a post-transform schema. User may
+          omit this argument and specify only `remove_fields` argument.
```

Comment:
```suggestion
          In case the user wants to only remove certain fields, the user may omit this argument and specify only `remove_fields` argument.
```

I believe this makes it slightly more clear.